pkg/didcomm/packer/packer.go

```diff
@@ -4,35 +4,45 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package crypto
+package packer
 
 import "github.com/hyperledger/aries-framework-go/pkg/kms"
 
-// Provider interface for Crypter ctx
-type Provider interface {
+// KMSProvider interface for Packer ctx
```

Comment:
are you sure this shouldn’t be a generic name?

Reply:
Fair point